management/Makefile

```diff
@@ -1,38 +1,29 @@
-
-# The kname of the context for the management cluster
-# The name of the context for your Kubeflow cluster
-NAME=$(shell yq r ./instance/settings.yaml name)
+NAME=$(shell yq r ./instance/Kptfile 'openAPI.definitions."io.k8s.cli.setters.name".x-k8s-cli.setter.value')
```

Comment:
nit: add a simple comment above, explaining Kptfile's values are set when calling kpt cfg set?

I think a goal of the makefile is to allow user customization. So it needs to be easy to understand.

Reply:
Right!

Reply:
Side Note: I ran the `kpt cfg set` with the default values to keep the values uniform throughout.